Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM silintl/php7:7.2
-MAINTAINER Mark Tompsett <mark_tompsett@sil.org>
+FROM php:7.3-apache-buster
+LABEL maintainer="Mark Tompsett <mark_tompsett@sil.org>"
```

Comment:
Nice, I wasn't aware that `MAINTAINER` is deprecated.

Reply:
Discovered it recently while reading the Dockerfile specification.